### DISCUSSION TITLE
How to open a second local window onto already open project

### DISCUSSION THREAD

**Discussion by jovlinger** [zed-industries/zed#12074] [Feature Requests]
I have a window open on my project . I would like a second window (for another monitor) to be open simultaneously. 

But I cannot. All attempts merely navigate back to the existing window. 

Three attempts that have failed:

1. I I open a new widnow and cmd-O and navigate to the project's root and open it, it warps back to the existing window.
2. If I open a new window and select the project with Enter, it warps back to the existing window.
3. If I open a new window and select the project with cmd-Enter, it warps back to the existing window.

All roads seem to lead back to the existing window. Help me obi-wan. What am I missing?

**Comment by jovlinger** [comment]
I was hoping the devs or community relations folks might chime in here.

**Comment by JosephTLyons** [comment]
Hey @jovlinger - I think you are right, multiple windows can't be opened pointing toward the same project. I tried looking for an issue requesting a change in the backlog, but didn't see one - it is possible I missed it.  Feel free to open one, so others can vote on it, and we can prioritize it accordingly based on interest.

**Reply by janek** [reply]
for others reading this: the issue seems to be https://github.com/zed-industries/zed/issues/9662, please upvote and optionally comment. There is also a duplicate at https://github.com/zed-industries/zed/issues/17618.

**Comment by stanleyjzheng** [comment]
My solution is to do `zed -n your_dir`. `zed -n` opens a new workspace even if you already have the same folder open, so that you can work with two windows of the same folder.

**Reply by kritonios01** [reply]
Yes this indeed works, but beware: if a file gets changed in the new window, you will have to either overwrite or discard the file in the first window.

**Reply by janek** [reply]
thanks a lot for this, works 
(thank you also for the warning @kritonios01)

**Reply by ennioVisco** [reply]
This doesn't seem to work with remote projects (WSL)

**Reply by OttoWRas** [reply]
This does not work with a remote connection using SSH either.
